mm2src/mm2_bitcoin/keys/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! Bitcoin keys.
 
-extern crate base58;
 extern crate bech32;
 extern crate bitcrypto as crypto;
+extern crate bs58;
```

Comment:
This isn't needed, we can also replace all the `extern crate` lines from this module, they are all rely on very old API from rustc.

Reply:
Turned out to be a big diff, let's do it in a different PR after the toolchain update is merged.